### PR TITLE
Make JSON output consistent across CLI commands

### PR DESCRIPTION
**Pull request by joestringer**
```release-note-minor
Consistently use `-o json` as the CLI arguments for printing JSON output across all commands that support JSON output
```

This PR refactors the JSON/MultipleOutput handling from `cilium/cmd/helpers` into `pkg/command`, then reuses this from various commands including `cilium-health`. While we're at it, remove the variations on json output like `--json` on a few commands, so all commands should be consistent with respect to how you get them to print JSON output.

Related: #2083

### CODE REVIEW

**Comment by joestringer** [comment]
test-me-please

File: cilium/cmd/bpf_policy_get.go [hunk 1]
```diff
@@ -37,10 +37,9 @@ var printIDs bool
 
 // bpfPolicyListCmd represents the bpf_policy_list command
 var bpfPolicyListCmd = &cobra.Command{
-	Use:     "list",
-	Aliases: []string{"get"},
```
Comment (ianvernon) [inline cilium/cmd/bpf_policy_get.go:41]: Please update `tests/helpers.bash` to change the invocation of this command, which is ran during the log-gathering scripts.

**Comment by joestringer** [comment]
test-me-please

File: cilium/cmd/bpf_policy_get.go (hunk 1 above)
Comment (joestringer) [reply cilium/cmd/bpf_policy_get.go:41]: Done.

**Comment by joestringer** [comment]
test-me-please

**Comment by joestringer** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**